src/page/home/interact/interact.tsx

```diff
@@ -0,0 +1,81 @@
+import type { FC } from 'react'
+import { useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import type { CardBackground } from '@/component/card/card'
+import classNames from 'classnames'
+import { Card } from '@/component/card/card'
+import './interact.scss'
+import '../i18n/index'
+
+type InteractCard = {
+  action: string
+  category: string
+  disabled: boolean
```

Comment:
`disabled` should be optional, so it will fe evaluated with `false` value  if undefined.